Formula/python@3.9.rb

```diff
@@ -84,12 +84,6 @@ class PythonAT39 < Formula
       sha256 "167e328cf68e9ec142f483fda9fafbb903be9a47dee2826614fdc24b2fbe6e06"
     end
 
-    # Further patch for Big Sur, remove in 3.9.2
-    # https://github.com/python/cpython/pull/23556
-    patch do
-      url "https://github.com/fxcoudert/cpython/commit/cd26ccbb.patch?full_index=1"
-      sha256 "59c98f991f839d610d53ca5c4af1464a99adf6c85e807f8a676b3e41c5dbe0a2"
-    end
```

Comment:
why does this get removed?

Reply:
Great Question! So what @fxcoudert actually does in that patch is 
explicitly force distutils to turn the mac version from an int into a string.

That only addresses the symptom of the problem and not the underlying
problem. The underlying problem is that ``MacOS.version`` as currently provided
by ruby in BigSur only has the base number and not the sub-version. Upon running
``./configure``, Python has some rules that it applies when retrieving variables set
in configure. One of those rules seems to be that pure number as environment
variables are treated as ints, not strings. This causes a problem when distutils trys
to part version and sub-version with ``string.split``. See [here](https://github.com/python/cpython/blob/0be3246d4f9c8eddcd55491901d95b09fe163f15/Lib/distutils/spawn.py#L101)

The patch is actually unecessary when you pass ``11.0`` as the MacOS version
to configure args explicitly, which is what I've done in this PR. If we were to
keep @fxcoudert's patch, you would actually be converting a string into a string
twice.

Reply:
I agree this shouldn't be removed.

Reply:
Ok. I brought it back. Either way it should still work.

Reply:
Ah, sorry, didn't see your reply when I posted my comment. Assuming FX's patch is accepted, this would continue to be the case regardless of whether or not the patch is removed here, right? (Future versions will have the modified code)

Reply:
So whether or not FX's patch is accepted, thing like Cython will still fail
to build along with any pip packages that use distutils.
My patch PR should all those issues.

Reply:
For clarity, Python is a brew package for c-based FFI in Python.